luigi/util.py

```diff
@@ -313,6 +316,41 @@ def clone_parents(_self, **kwargs):
 
         return task_that_inherits
 
+    def check_for_parameter_collisions(self, task_that_inherits):
+        error_msg = (
+            "Parameter name collision detected in tasks_to_inherit. "
+            "Parameter '{param}' in '{task}' duplicates "
+            "parameter '{param}' in {another_task}. "
```

Comment:
could probably be simplified to `Parameter {param} in {task} is duplicated in {another_task}`. No need to continually output the name of the parameter if there is a collision due to the names being the same.

Reply:
Agreed. Will make the change.